frontend/cypress/integration/featureFiles/workloads.feature

```diff
@@ -0,0 +1,60 @@
+Feature: Kiali Workloads page
+  
+  User opens the Workloads page and sees the bookinfo workloads.
+
+  Background:
+    Given user is at administrator perspective
+    And user is at the "workloads" page
+    And user selects the "bookinfo" namespace
+
+  @workloads-page
+  Scenario: See a table with correct info
+    Then user sees a table with headings
+      | Health | Name | Namespace | Type | Labels | Details
+    And the "details-v1" row is visible
+    And the health column on the "details-v1" row has a health icon
+    And the "Name" column on the "details-v1" row has a link ending in "/namespaces/bookinfo/workloads/details-v1"
+    And the "Namespace" column on the "details-v1" row has the text "bookinfo"
+    And the "Labels" column on the "details-v1" row has the text "app=details"
+    And the "Labels" column on the "details-v1" row has the text "version=v1"
+    And the "Type" column on the "details-v1" row has the text "Deployment"
+    And the "Details" column on the "details-v1" row is empty
```

Comment:
These are more "procedural" in style. The preferred syntax is probably less procedural: https://github.com/kiali/kiali/pull/5006#discussion_r859226118. That being said, I don't think it's a blocker and we can consolidate later.

Reply:
The Istio config page is probably a better example that is a "list" page: https://github.com/kiali/kiali/blob/master/frontend/cypress/integration/featureFiles/istio_config.feature

Reply:
@leandroberetta to be clear, you do not need to re-write these tests in the more procedural syntax. Just fix the false positive then these tests LGTM.

Reply:
ok, thanks for the clarification, I was looking in the example to pasted.